doc/index.rst

```diff
@@ -54,6 +54,9 @@ everything you need to know to use **PyMongo**.
 :doc:`developer/index`
   Developer guide for contributors to PyMongo.
 
+:doc:`common-issues`
```

Comment:
The new rst file still has to be included in the toctree at the bottom of the file.

Reply:
I think I have corrected it.